CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to FingerprintJS Pro Azure Integration
+
+## Working with code
+
+We prefer using [yarn](https://yarnpkg.com/) for installing dependencies and running scripts.
+
+For proposing changes, use the standard [pull request approach](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request). It's recommended to discuss fixes or new functionality in the Issues, first.
+
+* The `main` and `develop` branches are locked for the push action.
+* Releases are created from the `main` branch. If you have Azure Integration set up, it is running code from the `main` branch. Pull requests into the `main` branch are not accepted.
+* The `develop` branch functions as a candidate for the next release. Create your pull requests into this branch. The code in `develop` must always pass the tests.
+
+### How to build
+* After cloning the repository, run `yarn install` to install dependencies.
+
+* Run `yarn build` to build the project into the `dist` folder. The created `dist/fingerprintjs-pro-azure-function/fingerprintjs-pro-azure-function.js` and `dist/fingerprintjs-pro-azure-function-management/fingerprintjs-pro-azure-function-management.js` are meant to be deployed to Azure Functions.
+
+### How to deploy to Azure Functions
+
+1. Install the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) and sign in to Azure account according to the [Sign in with Azure CLI](https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli) guide.
+
+2. Install the [Azure Functions Core Tools](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=v4%2Cmacos%2Ccsharp%2Cportal%2Cbash#install-the-azure-functions-core-tools).
+
+3. Run
+    ```shell
+    func azure functionapp publish <AppFunctionName>
+    ```
+    where `<AppFunctionName> is a Azure Function App name in the Azure account.
+
+
+You can invoke the function from your local environment using the [start command](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=v4%2Cmacos%2Ccsharp%2Cportal%2Cbash#start) in Azure Functions Core Tools.
```

Comment:
Will this work for sure? For now, when I want to deploy the function app, first have to build the functions via `yarn run build`, and then `cd dist && func azure functionapp publish function-app-name`